portfolio/src/main/webapp/script.js

```diff
@@ -31,9 +31,19 @@ function addRandomFact() {
  * Adds a new Comment Box to the page
  */
 function newCommentBox(comment){
-  const commentBox = document.createElement('p');
+  // create comment elements
+  const commentBox = document.createElement('div');
+  const commentName = document.createElement('p');
+  const commentText = document.createElement('p');
+
+  // set text for elements
+  commentName.innerHTML = comment.name + ':';
+  commentText.innerHTML = comment.text;
```

Comment:
This is a security vulnerability by using innerHTML. I can post a comment that is HTML code and when someone else loads your site, it would execute that code. Use innertText instead.

https://www.reddit.com/r/learnjavascript/comments/9502x5/is_innerhtml_still_considered_bad/e3p31go/